rest/data/patch.go

```diff
@@ -100,10 +98,6 @@ func (p *DBPatchConnector) AbortPatchesFromPullRequest(event *github.PullRequest
 	err = model.AbortPatchesWithGithubPatchData(*event.PullRequest.ClosedAt,
 		owner, repo, *event.Number)
 	if err != nil {
-		grip.Error(message.Fields{
```

Comment:
why remove this?

Reply:
The route that calls this logs this failure in far greater detail.